katsdpingest/katsdpingest/ingest_session.py

```diff
@@ -1243,17 +1248,17 @@ def sensor_value(telstate, name):
         static_flag = np.uint8(1 << sigproc.IngestTemplate.flag_names.index('static'))
         cam_flag = np.uint8(1 << sigproc.IngestTemplate.flag_names.index('cam'))
         end_time = timestamp + self.cbf_attr['int_time']
-
         channel_slice = self.channel_ranges.input.asslice()
-        channel_mask = sensor_value(self.telstate_cbf, 'channel_mask')
-        if channel_mask is not None:
-            channel_mask = np.atleast_2d(channel_mask)
-            flags_in[:] = channel_mask[self.bls_channel_mask_idx, channel_slice].T * static_flag
+
+        channel_mask_sensor = sensor_value(self.telstate_cbf, 'channel_mask')
```

Comment:
So is the channel mask in telstate now an array of bool shaped (n_masks, n_channels)?

Reply:
Yes.